Makefile

```diff
@@ -71,6 +71,10 @@ java_tests:
 fintech-ui/node_modules:
 	cd fintech-examples/fintech-ui && npm install
 
+.PHONY : fintech-ui-test
```

Comment:
Probably we need to report coverage to codecov.io, not sure will it be picked automatically or not. But even if it does it is good idea to report coverage under separate label:
https://docs.codecov.io/docs/merging-reports#section-multiple-languages